### ISSUE TITLE
Any plans to make new release soon?

### ISSUE THREAD

**Issue by kloczek** [hercules-team/augeas#722]
I think that it would be good to flush currently committed changes and make new release as last releasease was two years ago :)

**Comment by kloczek** [comment]
ping .. :)

**Comment by waipeng** [comment]
Yes please we are also looking forward to a new release, for issues like #668 that has already been fixed

**Comment by kloczek** [comment]
Looks like 1.13.0 has been released however there is no dist tar ball on http://download.augeas.net/

**Comment by georgehansper** [comment]
As mentioned in #756, I've uploaded a tarball augeas-1.13.0.tar.gz to the [1.13..0 release](https://github.com/hercules-team/augeas/releases/tag/release-1.13.0)

**Comment by yuwenma** [comment]
Just FYI, if the Augeas team has plans to do this task, can we also have the docker build  update? Filed https://github.com/hercules-team/augeas/issues/769
